### ISSUE TITLE
Bildanpassung Inhaltsbanner Startseite

### ISSUE THREAD

**Issue by innosmith** [decidim-swiss/decidim.swiss#44]
Ist es nur mit CSS einfach möglich, Bilder für Inhaltsbanner auf der Startseite mit einem Farbverlauf weiss zu Bild oben und Bild zu weiss unten gemäss Screenshot anzupassen? Wenn die Bilder manuell mit dem Farbverlauf vorbereitet werden, sieht es je nach Bildschirmgrösse seltsam aus. Um den Effekt zu testen, könnte ein neuer Bildausschnitt mit den Bergen vom angehängten Unsplash-Bild verwendet werden.

Originalbild:
![denis-lekaj-9LHblUK3kg4-unsplash](https://user-images.githubusercontent.com/75796635/205328328-b9048238-c6cf-46d2-8f0d-ac00a55b7cc4.jpg)

Mockup:
![image002](https://user-images.githubusercontent.com/75796635/205328451-d1c3d51e-a24a-431d-98c3-252f6f26f367.jpg)

**Comment by carlobeltrame** [comment]
Ja, es ist möglich, aber auf alten Versionen von Safari wird es [möglicherweise nicht funktionieren](https://caniuse.com/mdn-css_properties_mask-image).

Ausserdem: Die Schrift auf diesem hellen Hintergrundbild ist sehr schlecht lesbar. Das könnte man mit einem leichten Schlagschatten der Schrift dramatisch verbessern. Wie fändest du das?

**Comment by innosmith** [comment]
Danke @carlobeltrame für deine Einschätzung und Vorschlag. Finde ich gut. Aufwand zur Umsetzung?

**Comment by carlobeltrame** [comment]
Wie besprochen, 2-4 Stunden. Tendenziell weniger, aber habe noch nicht viel mit gradients und masks gearbeitet, daher ist Unsicherheit drin.
